Sources/TuistSupport/Utils/DeveloperEnvironment.swift

```diff
@@ -36,6 +43,15 @@ public final class DeveloperEnvironment: DeveloperEnvironmenting {
         _derivedDataDirectory = location
         return location
     }
-
     // swiftlint:enable identifier_name
+    
+    @Atomic private var _architecture: MacArchitecture?
+    public var architecture: MacArchitecture {
+        if let _architecture = _architecture {
+            return _architecture
+        }
+        let output = try! System.shared.capture("/usr/bin/uname", "-m").chomp()
```

Comment:
In Swift 5.5, it's actually possible to mark the getter throws, so we can omit the bang here. But that's just for the future reference 🙂

Reply:
Oh, I didn't know that. I think for this particular method we want to abort the execution. Otherwise, we'll end up adding unnecessary `try` and `throws` upstream.